src/nvim/message.c

```diff
@@ -1362,7 +1366,10 @@ void msg_start(void)
     msg_col =
       cmdmsg_rl ? Columns - 1 :
       0;
-  } else if (msg_didout) {                // start message on next line
+    if (none_msg_area && ccline.cmdprompt == NULL) {
```

Comment:
It looks like `ccline` was only made global so it could be accessed here. Could be make `ccline` unglobal and just provide some getter function instead?

Reply:
OK.